scene/gui/line_edit.cpp

```diff
@@ -369,6 +371,11 @@ void LineEdit::gui_input(const Ref<InputEvent> &p_event) {
 				selection_fill_at_caret();
 			}
 		}
+
+		if (drag_action && Input::get_singleton()->get_current_cursor_shape() == Input::CURSOR_CAN_DROP) {
```

Comment:
Checking cursor shape isn't probably very reliable. It can be changed via script etc.
Not sure how it could be done better though. Can't you just use `drop_data()`?

Reply:
I've changed the code to use
`get_viewport()->gui_get_drag_data()`
for the check.